pyaedt/hfss.py

```diff
@@ -6158,7 +6185,8 @@ def wave_port(
         reference : int, str, list or :class:`pyaedt.modeler.cad.object3d.Object3d`
             Ending object for the integration line or reference for Terminal solution. Can be multiple objects.
         create_port_sheet : bool, optional
-            Whether to create a port sheet or use given start_object as port shee.
+            Whether to create a port sheet or use given start_object as the surface to create
```

Comment:
```suggestion
            Whether to create a port sheet or use given starting object as the surface to create
```
Again, description should use English versus code whenever possible.

Reply:
updated